app.js

```diff
@@ -0,0 +1,205 @@
+1 task
+
+ function sayHello(name) {
+    if (name === "Mark") {
+        return "Hi," + " " + name + "!";
+    }
+    else {
+        return "Hello," + " " + name + "!";
+    }
+}
+console.log (sayHello("Oleg"));
+console.log (sayHello("Viktor"));
+console.log (sayHello("Mark"));
+
+
+const sayHello=(name) => {
+    if (name === "Mark") {
+        return "Hi," + " " + name + "!";
+    }
+    else {
+        return "Hello," + " " + name + "!";
+    }
+}
+console.log (sayHello("Oleg"));
+console.log (sayHello("Viktor"));
+console.log (sayHello("Mark"));
+
+
+
+
+
+2 task
+function pythagorean(sideM, sideN) {
+    return Math.sqrt(Math.pow (sideM, 2) + Math.pow (sideN, 2));
+}
+console.log (pythagorean (4, 3));
+
+const pythagorean = (sideM, sideN) => {
+    return Math.sqrt(Math.pow(sideM, 2) + Math.pow(sideN, 2));
+}
+console.log (pythagorean(4, 3)); 
+
+
+
+3 task
+
+function min(a, b) {
+    if (a >= b) {
+        return b;
+    }
+    else {
+        return a;
+    }
+}
+console.log (min (2, 5));
+console.log (min (3, -1));
+console.log (min (1, 1));
+
+const min = (a, b) => {
+    if (a >= b) {
+        return b;
+    }
+    else {
+        return a;
+    }
+}
+console.log (min (2, 5));
+console.log (min (3, -1));
+console.log (min (1, 1));
+
+
+
+
+
+
+4 task 
+
+function isEven (n) {
+    if(!(n % 2)) {
+        return "true";
+}
+    else {
+        return "falce";
+    }
+} 
+console.log (isEven(5));
+console.log (isEven(4));
+
+const isEven = (n) => {
+    if(!(n % 2)) {
+        return "true";
+    }
+    else {
+        return "falce";
+    }
+}
+console.log (isEven(5));
+console.log (isEven(4));
+console.log (isEven(95));
+
+
+
+
+
+
+
+
+task 5
+
+function deleteChars(str) {
+    const someString = str.slice(1, -1);
+    return someString;
+}
+console.log(deleteChars("Hello"));
+console.log(deleteChars("A"));
+
+const deleteChars = (str) => {
+    const someString = str.slice(1, -1);
+    return someString;
+}
+console.log(deleteChars("Hello"));
+console.log(deleteChars("A"));
+
+
+
+
+
+
+
+
+task 6 
+
+function someFn(string) {
+    return string.charAt(0).toUpperCase(1) + string.slice(1).toLowerCase(-1);

```

Comment:
`toUpperCase()` and `toLowerCase()` don't accept any arguments, that's why if you remove 1 and - 1 from those methods everything will work fine as previously